demos/JWST/S2_NIRISS_fs_template

```diff
@@ -0,0 +1,41 @@
+# Eureka! Control File for Stage 2: Data Reduction
+
+# Stage 2 Documentation: https://eurekadocs.readthedocs.io/en/latest/ecf.html#stage-2
+
+suffix                  rateints    # Data file suffix
+
+# Controls the cross-dispersion extraction
+slit_y_low              -1      # Use None to rely on the default parameters
+slit_y_high             50      # Use None to rely on the default parameters
+
+# Modify the existing file to change the dispersion extraction - FIX: DOES NOT WORK CURRENTLY
+wave_orders         [1,2,3]     # Which orders to get wavelengths for
+
+# Note: different instruments and modes will use different steps by default
+skip_bkg_subtract       False   # Not run for TSO observations
+skip_imprint_subtract   True    # Not run for NIRSpec Fixed Slit
```

Comment:
Change comments on Lines 16-29 to reference NIRISS.
Double check to make sure these True/False settings are correct for NIRISS.